modules/ROOT/pages/endpoint-sftp-receive-target.adoc

```diff
@@ -48,13 +48,27 @@ Create SFTP receive endpoints by configuring these settings on the *<host>* or *
 | Port number of the SFTP server.
 | x
 
-| Username
-| Username for the SFTP server.
-| x (if the server is authenticated)
+| Authentication type
+a| Authenication method used to secure the SFTP endpoint. Values are: 
 
-| Password
-| Password for the SFTP server.
-| x (if the server is authenticated)
+* *None*
++
+Provides no authentication for the endpoint
++
+* *Basic*
++
+Authenticates by using a username and password
++
+* *Identity Key*
++
+Authenticates by using a private SSH identity key
++
+* *Basic and Identity Key*
++
+Authenticates by using a username, password, and public SSH identity key
+| x
+
+include::partial$sftp-endpoint-authentication-partial.adoc[]
```

Comment:
Is this partial supposed to be in this cell? The cell is for "Required"

Reply:
This renders fine. It adds new rows to the table.